WordPress/Classes/ViewRelated/Media/Tenor/TenorPageable.swift

```diff
@@ -0,0 +1,41 @@
+
+struct TenorPageable: Pageable {
+    let itemsPerPage: Int
+    let position: String?
+    let currentPageIndex: Int
+
+    static let defaultPageSize = 36 // 6 rows x 6 columns on iPad Landscape, up to 9 rows x 4 cols on iPhone
```

Comment:
> 6 rows x 6 columns on iPad Landscape, up to 9 rows x 4 cols on iPhone

Is this always true? I'm wondering why this is `36`, but Giphy and Stock Photos are `40`.

Reply:
Hi @ScoutHarris thanks for the review. I chose 36 because that will fit for the first visible page of the picker on both iPad (10.5″) (I didn't take the iPad 12" into consideration) and on any iPhone. Please take a look at screenshots posted here https://wp.me/pbMoDN-hD.

Reply:
> I didn't take the iPad 12" into consideration

I'm curious as to why not? We do support this device. And if you do, (if I understand correctly) wouldn't that explain the other two pickers being 40, and why then Tenor should as well?

Reply:
Well in iPad 12.9" portrait, the first visible page take 8 rows x 6 cols = 48 items. I didn't take it into consideration as I owned an analytics data where there is only 2% out of 3M devices are iPad 12". That couldn't be the same here though, but with 40 or 36, on the Pro two requests will be issued any way (the Tenor also has a maximum 50 images per page). Any way, I can put it back to 40 as other pickers, what do you think?

Reply:
Thanks for the explanation @ngoctr .

We don't support specific _devices_ per se, but iOS versions. Basically, if it can run iOS 11+, then we support it. 

Regarding which default to use (40 or 48) I'll leave that up to you, whichever works best for Tenor. We don't have to match the others if that's not what's best. I suspect it will be more obvious from my perspective when the UI is in place and testable.

Reply:
Thank you very much @ScoutHarris. I think it would be (better) consistent, to keep it as 40 as other pickers. I agree that we don't have any UI in this PR so it is really not obvious to evaluate.